gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_5_0/05_add_integration_groups_table.yml

```diff
@@ -0,0 +1,14 @@
+databaseChangeLog:
+    - changeSet:
+          id: 4.5.0_add_integration_groups
+          author: GraviteeSource Team
+          changes:
+              - createTable:
+                    tableName: ${gravitee_prefix}integrations_groups
```

Comment:
I'm not so sure about this naming 🤔 @jgiovaresco @michel-barret wdyt?

Reply:
If we follow what is done for APIs and Applications it should be `${gravitee_prefix}integration_groups`